src/test/java/com/ioet/bpm/organizations/BpmOrganizationsApiApplicationTests.java

```diff
@@ -0,0 +1,135 @@
+package com.ioet.bpm.organizations;
+
+import com.ioet.bpm.organizations.boundaries.OrganizationController;
+import com.ioet.bpm.organizations.domain.Organization;
+import com.ioet.bpm.organizations.repositories.OrganizationRepository;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.util.Optional;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class BpmOrganizationsApiApplicationTests {
+
```

Comment:
All the tests need to be reviewed.

Reply:
@enriquezrene Please Check the new tests and ask me if these are correct.